### PR TITLE
Easier testing

### PR DESCRIPTION
**Pull request by lomignet**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -69,38 +71,60 @@ test: unittest pep8
...
+	@echo Built Pydundas version $(PYDUNDASVER).
+
+getver:
+PYDUNDASVER := $(shell $(PY3) -c 'from pydundas import __version__ as v; print(v)')
 
 ## Upload
 
+pypiversion: getver
+	@echo Module version: $(PYDUNDASVER).
+	@echo Latest version on test.pypi: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
```
Comment (cdemeijer) [inline Makefile:88]: You're making assumptions about the availability of curl and jq here. Don't really think it is a problem per se, but you might want to make it explicit.

File: Makefile [hunk 2]
```diff
@@ -69,38 +71,60 @@ test: unittest pep8
...
 	# --repository-url https://test.pypi.org/legacy/
+	@echo Latest version of Pydundas on test.pypi before upload: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
 	$(PY3) -m twine upload --repository testpypi dist/*
+	@echo Latest version of Pydundas on test.pypi after upload: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
+	@echo Note that it can take a few minutes to get the new version available. Check with make pypiversions.
 
 # Yes, test test: test the install from the test pypi repo.
-testtestinstall:
+testtestinstall: getver
 	# Install pydundas from tespypi
```
Comment (cdemeijer) [inline Makefile]: Working so much with tez that you can't spell 'test' properly anymore? :stuck_out_tongue_winking_eye:

File: Makefile [hunk 3]
```diff
@@ -69,38 +71,60 @@ test: unittest pep8
...
 	rm -rf $(TESTENV)
 	# -m venv is always there, conda maybe not
 	python3 -m venv $(TESTENV)
 	# Note: cd first to prevent the current directory to be found as valid module.
 	cd $(TESTENV) && $(TESTENV)/bin/python3 -m pip install --upgrade --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pydundas
-	$(TESTENV)/bin/pip freeze | grep -i pydundas
+	@echo Module version: $(PYDUNDASVER).
+	@echo Version on test Pypi: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
+	@echo Version in test env: $(shell $(TESTENV)/bin/pip freeze | grep -i pydundas).
+	# Should not error out
```
Comment (cdemeijer) [inline Makefile]: So.., there is no need for an assertion? Just a lack of exceptions is ok? Is that the python way?

**Review by cdemeijer — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (lomignet) [reply Makefile:88]: It's only for the owner of the pydundas pypi account, so it's OK indeed.

File: Makefile (hunk 3 above)
Comment (lomignet) [reply Makefile]: It's not a proper unit test, it's a quick and dirty oneliner inside a makefile. 
Real tests are inside the `tests`  directory and are more like what you expect.

**Comment by lomignet** [comment]
Failing checks are due to wrong semaphore setup - all is good.

**Pull request merged**